client/src/components/Footer.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Link } from "react-router-dom";
+import energy from "../assets/energy.svg";
+import news from "../assets/news.svg";
+import car from "../assets/car.svg";
+import options from "../assets/options.svg";
+
+const FooterContainer = styled.div`
```

Comment:
Semantically speaking, it might me more appropriate to use ```footer``` instead of ```div```, I guess, but in my case I used directly a nav, like you did below, without any footer or div wrapping it.